docs/canvas/canvas-function-reference.asciidoc

```diff
@@ -1,17 +1,2133 @@
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_
+
+Alias: `column`
+|`string`
+|The column to sort on. If column is not specified, the `datatable` is sorted on 
+the first column.
+
+|`reverse`
+|`boolean`
+|Reverse the sort order. If reverse is not specified, the `datatable` is sorted in ascending order. 
+
+Default: `false`
+|===
+
+*Returns:* `datatable`
+
+
+[float]
+[[staticColumn_fn]]
+=== `staticColumn`
+
+Adds a column with a static value. See also <<alterColumn_fn>> and <<mapColumn_fn>>.
+
+*Accepts:* `datatable`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `column`, `name`
+|`string`
+|The name of the new column
+
+|`value`
+|`string`, `number`, `boolean`, `null`
+|The value to insert in each column. Tip: Use a sub-expression to rollup other columns into a static value. 	 
+
+Default: `null`
+|===
+
+*Returns:* `datatable`
+
+
+[float]
+[[string_fn]]
+=== `string`
+
+Outputs a string made of other strings. Mostly useful when combined with 
+sub-expressions that output a string, or something castable to `string`.
+
+*Accepts:* `null`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** *†*
+
+Alias: `value`
+|`string`
+|One or more `strings` to join together, include spaces where needed
+|===
+
+*Returns:* `string`
+
+
+[float]
+[[switch_fn]]
+=== `switch`
+
+Performs conditional logic with multiple conditions. See also <<case_fn>>,
+which builds a `case` to pass to the <<switch_fn>> function.
+
+*Accepts:* `any`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** *†*
+
+Alias: `case`
+|`case`
+|The list of conditions to check. See <<case_fn>>
+
+|`default` 
+
+Alias: `finally`
+|`any`
+|The default returned value, if no cases match
+|===
+
+*Returns:* Depends on your input and arguments
+
+
+[float]
+[[t_fns]]
+== T
+[[table_fn]]
+=== `table`
+
+Configures a data table element.
+
+*Accepts:* `datatable`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|`font`
+|`style`
+|Font style 
+
+Default: `{<<font_fn>>}`
+
+|`paginate`
+|`boolean`
+|Show pagination controls. If set to `false` only the first 
+page is displayed.  
+
+Default: `true`
+
+|`perPage`
+|`number`
+|Show this many rows per page 
+
+Default: `10`
+
+|`showHeader`
+|`boolean`
+|Show or hide the header row with titles for each column 
+
+Default: `true`
+|===
+
+*Returns:* `render`
+
+
+[float]
+[[tail_fn]]
+=== `tail`
+
+Gets the last N rows from the end of a `datatable`. See also <<head_fn>>
+
+*Accepts:* `datatable`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `count`
+|`number`
+|Return this many rows from the end of the `datatable`
+|===
+
+*Returns:* `datatable`
+
+
+[float]
+[[timefilter_fn]]
+=== `timefilter`
+
+Creates a time filter for querying a source.
+
+*Accepts:* `filter`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|`column` 
+
+Alias: `field`, `c`
+|`string`
+|The column or field to attach the filter to 
```

Comment:
I would avoid ending a sentence with `to`. How about:

The column or field to attach to the filter.